src/BinDeps.jl

```diff
@@ -1,37 +1,32 @@
 module BinDeps
     importall Base
 
+    using Compat
+
     export @make_run, @build_steps, find_library, download_cmd, unpack_cmd,
             Choice, Choices, CCompile, FileDownloader, FileRule,
             ChangeDirectory, FileDownloader, FileUnpacker, prepare_src,
             autotools_install, CreateDirectory, MakeTargets, SystemLibInstall
 
-    if VERSION >= v"0.4.0-dev+3844"
-        import Base.Libdl: dlext, dlpath, RTLD_LAZY, DL_LOAD_PATH
-    else
-        const dlext = isdefined(Base.Sys, :shlib_ext) ? Base.Sys.shlib_ext : Base.Sys.dlext # Julia 0.2/0.3 compatibility
```

Comment:
As I noted in #137, support for 0.2 was already broken because `Sys.dlpath` does not exist in 0.2.   So, this is not a regression, and I don't think it should hold up the patch.

(See also JuliaLang/Compat.jl#60)